api-docs/rst/dev-guide/concepts.rst

```diff
@@ -639,10 +639,17 @@ and how many resources to delete.
 When deleting servers, Autoscale follows these rules:
 
 -  If no new servers are in the process of being built, the oldest
-   servers are deleted first.
+   servers are chosen to be deleted first.
 
 -  If new servers are in the process of being built and in a "pending"
-   state, these servers are deleted first.
+   state, these servers are chosen to be deleted first.
+
+-  After choosing the servers to delete autoscale will put the server's
+   corresponding load balancer's node in DRAINING **if** the group launch
+   configuration has cloud load balancer and ``draining_timeout`` in it.
+   The server will be deleted after ``draining_timeout`` has expired. If the
+   group launch configuration doesn't have cloud load balancer and ``draining_timeout``
+   then server is just deleted
```

Comment:
After selecting servers for deletion, the Autoscale process deletes each server immediately, unless the server has an associated load balancer that has been configured with a draining timeout period.  In these cases, Autoscale deletes the server after the draining timeout period for the load balancer expires. 

fter selecting the servers to delete, the Autoscale process puts any load balancers associated with the servers in DRAINING mode if the group launch configuration for the server includes the ``draining_timeout`` parameter. If the server A server is deleted after the draining timeout period expires. 
.